eventonica-api/routes/events.js

```diff
@@ -0,0 +1,69 @@
+const express = require('express');
+const cors = require('cors');
+const router = express.Router();
+const Event = require('../models/event');
+const { Op } = require("sequelize");
+
+router.use(express.json());
+router.use(cors());
+router.use(express.urlencoded({ extended: true }))
+
+//GET list of events
+router.get('/', (req, res) => {
+  Event.findAll()
+  .then(events => res.json(events))
+  .catch(error => {
+    console.log('ERROR:', error)
+  });
+});
+
+//ADD new event
+router.post('/add', (req, res) => {
+  const newEvent = req.body;
```

Comment:
Use [array destructuring](https://github.com/airbnb/javascript#destructuring). This recommendation comes from the airbnb style guide. I mentioned using linters last time, but here's a concrete example of how your code would look different if you were applying linting rules.

I admit styles will vary from team to team, but I like the Airbnb style guide as a starting point and that's what I've used on my last two teams.

So this would look like:

```
const {body: newEvent} = req;
```